### DISCUSSION TITLE
How to force the windows application to light mode

### DISCUSSION THREAD

**Discussion by taduyde** [microsoft/react-native-windows#10372] [Q&A] [answered]
How to force the windows application to light mode?

**Comment by chrisglein** [comment, marked as answer]
For UWP apps you set that on the `Application` object in the XAML like so: https://social.msdn.microsoft.com/Forums/vstudio/en-US/5baadd1f-63e0-4b6b-8f9c-87b799f17ca4/uwp-forcing-a-theme-for-my-application-dark-or-light?forum=wpdevelop.
For an RNW app, you'll need to delve into your app template and make that same edit it the XAML that was generated for you.

**Reply by taduyde** [reply]
Thank you, it's worked
